zerobuf/share/zerobuf/CMake/zerobufGenerateCxx.cmake

```diff
@@ -14,7 +14,7 @@ function(zerobuf_generate_cxx Name OutputDir)
   foreach(FILE ${ARGN})
     get_filename_component(ZEROBUF_OUTPUT ${FILE} NAME_WE)
     set(ZEROBUF_HEADER "${OutputDir}/${ZEROBUF_OUTPUT}.h")
-    set(ZEROBUF_SOURCE "${OutputDir}/${ZEROBUF_OUTPUT}.cpp")
+    set(ZEROBUF_SOURCE "${OutputDir}/${ZEROBUF_OUTPUT}.hpp")
```

Comment:
Is a source kind of. Should be ipp actually. The both should be public header anyways?